src/components/AccountsPage/AccountList/Header.js

```diff
@@ -72,9 +71,9 @@ class Header extends PureComponent<Props, { focused: boolean }> {
           onChange={onTextChange}
           value={search}
         />
-        <Text color="dark" ff="Museo Sans" fontSize={6} data-e2e="dashboard_AccountCount">
+        <Box>
```

Comment:
is this wrapping required or can Box be dropped?

Reply:
indeed, no need for box here